webknossos/webknossos/dataset/properties.py

```diff
@@ -278,7 +281,7 @@ def __layer_properties_pre_structure(
                 mag_view_properties_pre_unstructure(m) for m in d["wkwResolutions"]
             ]
             del d["wkwResolutions"]
-        obj = converter_fn(d)
+        obj = converter_fn(d, _type)
```

Comment:
since `_type` is used now, I'd rename it to `type`.

Reply:
`type` is a keyword/builtin function in Python. We could rename it to `dtype`, but that typically implies `float` or `uInt8`or similar to me. #namingthings

Open to suggestions.

Reply:
ah, I see. maybe `current_type` or `type_value`. I don't have a strong opinion.